apps/admin/src/app/providers/i18nProvider.ts

```diff
@@ -1,8 +1,202 @@
+import { TranslationMessages } from 'ra-core'
```

Comment:
```suggestion
import type { TranslationMessages } from 'ra-core'
```

Reply:
Weird it doesn't throw an error. The EsLint rule is in the config.